File: Unit 7/Unit7B.R
# Set filepath
dataFile <- file.path("C:\\Users\\laure\\OneDrive\\",
                      "Documents\\Purdue\\IN300 - Programming for Data Analysis",
                      "\\Unit 7\\IN300_Dataset2.txt")

# Parse data
data <- scan(dataFile)

# Create 2 dimensional matrix
my2dmatrix <- matrix(data, nrow=2500, ncol=100)

print(my2dmatrix)

# Create 3 dimensional array
my3dArray <- array(data, dim=c(250, 10, 10))

print(my3dArray)

# Slice 2d matrix
print(my2dmatrix[,2:5])

File: Unit 7/Unit7B.py
import numpy as np

data = np.loadtxt("C:\\Users\\laure\\OneDrive\\Documents\\Purdue\\IN300 - Programming for Data Analysis\\Unit 7\\IN300_Dataset2.txt")

# Reshape data into a 2d matrix
matrix_2d = data.reshape(2500, 100)

print(matrix_2d)

# Reshape into a 3d matrix
matrix_3d = data.reshape(2500, 10, 10)

print(matrix_3d)

# Slice the matrix
slice = matrix_2d[2:5:10, :]

print(slice)
